CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.6
+
+- Supporting [Regexp query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-regexp-query.html)
```

Comment:
Could you please move this URL to the `_query.dart` file, and link only to the PR here? Make it similar to the previous ones...

Reply:
fixed 🔧